packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimeRangeContent.tsx

```diff
@@ -163,6 +195,27 @@ export const TimeRangeContent = (props: Props) => {
         <Trans i18nKey="time-picker.range-content.apply-button">Apply time range</Trans>
       </Button>
 
+      <div className={style.buttonsContainer}>
+        <Button
+          aria-label={t('time-picker.copy-paste.tooltip-copy', 'Copy time range to clipboard')}
```

Comment:
as mentioned, i don't think you need `aria-label` if you've specified a `tooltip`

Reply:
ah i didn't meet the first condition, i just looked at the button itself without triggering the tooltip. i've removed the `aria-label` and tested with VoiceOver and it works as expected. thanks for double-checking this! :)